debian/libmiral4.symbols

```diff
@@ -393,5 +393,6 @@ libmiral.so.4 libmiral4 #MINVER#
  MIRAL_3.1@MIRAL_3.1 3.1.0
  (c++)"miral::WaylandExtensions::zwlr_foreign_toplevel_manager_v1@MIRAL_3.1" 3.1.0
  MIRAL_3.2@MIRAL_3.2 3.2.0
- (c++)"miral::Output::logical_group_id()@MIRAL_3.2" 3.2.0
```

Comment:
What happened here, why is this dropped?

Reply:
Because it caused FTBFS on my system.

I'm not sure why it was there (I don't think we ever had that symbol) and I don't know why builds ever worked with it listed as required.